### DISCUSSION TITLE
Change Cursor Style

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-app#2300] [Q&A] [answered]
How to change the cursor style to underline or beam I mean I know the code but the problem is where should I put it to effect the changes ?

echo -e '\e[2 q' # Change to block 
echo -e '\e[4 q' # Change to underline 
echo -e '\e[6 q' # Change to bar

But this is temporary on restarting the app the modified cursor style is gone. Where should I put it to make it permanent.

Regards.

**Comment by agnostic-apollo** [comment, marked as answer]
Check Changelog at https://github.com/termux/termux-app/releases/tag/v0.114 and https://wiki.termux.com/wiki/Terminal_Settings

**Reply by ghost** [reply]
Thanks 😊 I got it
